app/styles/components/gleam-logo.css

```diff
@@ -0,0 +1,12 @@
+.gleam-logo-container {
+  display: block;
+  width: 100%;
+  height: 100%;
+}
+
+.gleam-logo-container canvas {
+  display: block;
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
```

Comment:
We're setting the same styles via JS, why are we duplicating here?

Reply:
I really should learn to remove my debugs better sorry!